src/components/shared/ResultList.css

```diff
@@ -113,10 +113,12 @@
   word-break: break-all;
   /** https://searchfox.org/mozilla-central/source/devtools/client/themes/variables.css **/
   color: var(--grey-40);
-  margin-left: 15px;
+  margin-left: 5px;
   text-overflow: ellipsis;
   overflow: hidden;
   white-space: nowrap;
+  direction: rtl; /* make ellipsis on the beginning of a string */
```

Comment:
This is a solid trick but I'm wondering if @yzen could weigh in on accessibility with it.

Reply:
It seems to have no negative effect on screen readers. I think this should be fine.